core/src/test/java/io/opencensus/stats/ViewDataTest.java

```diff
@@ -197,22 +199,22 @@ public void preventStartTimeLaterThanEndTime() {
 
   private static final BucketBoundaries BUCKET_BOUNDARIES = BucketBoundaries.create(
       Arrays.asList(10.0, 20.0, 30.0, 40.0));
-  
-  private static final Mean MEAN = Mean.create();
 
-  private static final ImmutableMap<List<TagValueString>, MeanData> ENTRIES =
+  private static final Aggregation DISTRIBUTION = Distribution.create(BUCKET_BOUNDARIES);
+  
+  private static final ImmutableMap<List<TagValueString>, DistributionData> ENTRIES =
       ImmutableMap.of(
           Arrays.asList(V1, V2),
-          MeanData.create(0, 1),
+          DistributionData.create(1, 1, 1, 1, 0, 0, 1, 0),
           Arrays.asList(V10, V20),
-          MeanData.create(50, 2));
+          DistributionData.create(1, 1, 1, 1, 0, 0, 1, 0));
```

Comment:
Should we use two different AggregationDatas, as before?

Reply:
Sure, updated.